model/translation_request.go

```diff
@@ -83,6 +84,15 @@ func NewTranslationStatusFromReader(reader io.Reader) (*TranslationStatus, error
 	return &status, nil
 }
 
+func NewTranslationStatusFromBytes(data []byte) (*TranslationStatus, error) {
+	var status TranslationStatus
+	err := json.Unmarshal(data, &status)
+	if err != nil && err != io.EOF {
```

Comment:
I think `err != io.EOF` is not relevant anymore.

Reply:
Honestly I'd go with `if err != nil`, but I saw it was being used in other bytes parsers. I've only experienced `io.EOF` when the JSON payload is incomplete, so I'm not sure why we would ignore it. cc: @gabrieljackson do you know something about this?

Reply:
I think `err != io.EOF` is used with decoder. But when we already have a valid `[]byte`, we will not encounter `EOF` anymore.

Reply:
Yeah, you're right there. I may remove that check everywhere then.